### PR TITLE
Add pkgproj's for openSUSE 42.1 and Ubuntu 16.10

### PR DESCRIPTION
**Pull request by mellinoe**
@ericstj 

The official build is already running for these distros. Once merged, we should get published packages in the next build.

### CODE REVIEW

**Comment by ericstj** [comment]
I did a cursory look and it looks ok.  @gkhanna79 and @dagood should review as well.

File: src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/opensuse/42.1/Microsoft.NETCore.Runtime.CoreCLR.pkgproj [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <NativeSplittableBinary Include="$(BinDir)libmscordaccore.so" />
+    <NativeSplittableBinary Include="$(BinDir)libmscordbi.so" />
+    <NativeSplittableBinary Include="$(BinDir)libsos.so" />
+    <NativeSplittableBinary Include="$(BinDir)libsosplugin.so" />
+    <NativeSplittableBinary Include="$(BinDir)System.Globalization.Native.so" />
+    <ArchitectureSpecificNativeFile Include="$(BinDir)sosdocsunix.txt" />
+    <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll" />
+    <ArchitectureSpecificNativeFile Include="$(BinDir)System.Private.CoreLib.ni.dll" />
+    <ArchitectureSpecificLibFile Include="$(BinDir)System.Private.CoreLib.dll" />
+    <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll" />
```
Comment (dagood) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/opensuse/42.1/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: In opensuse 13.2 there's a line `<ArchitectureSpecificLibFile Include="$(BinDir)SOS.NETCore.dll" />` here, is that intentionally not here? (I don't know what this file is, but maybe it was added after you started this change?)

Comment (dagood) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/opensuse/42.1/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: I see that `SOS.NETCore.dll` is in the new ubuntu package, though.

**Comment by dagood** [comment]
LGTM, just wondering about that one file.

File: src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/opensuse/42.1/Microsoft.NETCore.Runtime.CoreCLR.pkgproj (hunk 1 above)
Comment (mellinoe) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/opensuse/42.1/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: Good catch, I must have copied this file before pulling my branch up to master, but then did that before copying the Ubuntu 16.10 one.

I went through all of the openSUSE files I copied and made sure they were up to date. This looked like the only error.

**Comment by gkhanna79** [comment]
@RussKeldorph PTAL

**Comment by mellinoe** [comment]
@RussKeldorph Any objections here?

**Comment by mellinoe** [comment]
I'm going to go ahead and merge this. We already have native corefx packages available for these RIDs, and after this they will be fully supported.

**Pull request merged**